ci2/ci/build.py

```diff
@@ -448,7 +504,10 @@ def config(self):
                                           service_account_name='ci2-agent',
                                           parent_ids=self.deps_parent_ids())
```

Comment:
Does this mean we're not going to include namespaces in the yaml files? Namespaces are always defined in the build configuration?

Reply:
I'm not quite sure which code you're referring to.

Correct.  Namespaces should not be able to be referred to directly.  runImage is run by batch, so executes in batch-pods.  deploy takes yaml, but requires a namespace which I verify is was created by a `createNamespace` step, see `get_namespace`.  My longer term intention is to lock down the yaml files by whitelisting k8s features and parsing/validating them.  Right now, I explicitly create them with `kubectl -n <namespace>` which restricts itself to a namespace according to the documentation (although I don't know exactly what that means).

Does that answer your question?